src/components/ErrorBar/ErrorBar.tsx

```diff
@@ -0,0 +1,19 @@
+import { Snackbar } from 'material-ui'
+import * as React from 'react'
+import './ErrorBar.css'
+
+type Props = {
+  message: string | null
+  onClose(): void
+}
+
+const THIRTY_SECONDS = 30000
```

Comment:
Maybe a bit long?

Reply:
Try it out - easy to change if you don't like it.